### DISCUSSION TITLE
Create a pause of pH and ORP dosing for a set period of time while in pool.

### DISCUSSION THREAD

**Discussion by kennybitte** [tagyoureit/nodejs-poolController#825] [Ideas]
It would be great if there was a way to pause chemical dosing when you were in the pool so you wouldn't be sprayed with a higher than normal level of muriatic acid or chlorine from deck jets or other water features.

**Comment by rstrouse** [comment]
Normally that isn't really a problem unless you are dosing a spa.  However, you can set a mixing time which will pause the dosing schedule and then resume it after the mixing time expires.
